CMakeLists.txt

```diff
@@ -62,39 +62,44 @@ if(ENABLE_TESTING)
   add_subdirectory(Tests)
 endif()
 
+set(swift_lib_dir "lib/swift")
+
 if(NOT BUILD_SHARED_LIBS)
+  set(swift_lib_dir "lib/swift_static")
+  
   set_property(GLOBAL APPEND PROPERTY Foundation_EXPORTS
     CoreFoundation CFXMLInterface CFURLSessionInterface)
   install(TARGETS CoreFoundation CFXMLInterface CFURLSessionInterface
-    DESTINATION lib/swift_static/$<LOWER_CASE:${CMAKE_SYSTEM_NAME}>)
+    DESTINATION ${swift_lib_dir}/$<LOWER_CASE:${CMAKE_SYSTEM_NAME}>)
```

Comment:
This seems incorrect.  `CoreFoundation` is always built statically for Foundation, so this should always be `swift_static` right?

Reply:
Good catch!

Reply:
Oh, actually this is still correct because it will only be installed if we are compiling statically, in which case the variable is already set to `lib/swift_static`. I can change it back if you want.

Reply:
That would be great.